website/pages/en/_meta.js

```diff
@@ -10,6 +10,7 @@ export default {
   glossary: '',
   'arbitrum-faq': '',
   tokenomics: 'Tokenomics',
+  'graph-client': 'Graph Client',
```

Comment:
does it the right place in the sidebar for Graph Client docs?

Reply:
I think this should be part of Querying section

Reply:
moved!